### DISCUSSION TITLE
Windows indexing question

### DISCUSSION THREAD

**Discussion by arbateman** [Flow-Launcher/Flow.Launcher#2572] [Q&A] [answered]
I have been using Everything as my windows search. Recently I disabled the windows indexing service. Everything worked fine. I can't seem to get Flow Launcher to search local files. It says the files are not indexed. I do have Everything selected inside Flow Launcher.  I have tried a few combos of having Flow Launcher, Everything, and Everything Tool bar installed. Ideally I would like the latest Flow Launcher to be the only thing I need.  Am I missing something?

**Comment by taooceros** [comment]
Check the setting to switch to everything search for explorer.

**Reply by arbateman** [reply]
I did that.  It still says it needs indexing.

**Reply by taooceros** [reply]
Mind paste a screenshot?

**Reply by arbateman** [reply]
The top screenshot is when I use the hotkey atl-spacebar. The bottom is when I click the Flow Launch icon that I have pinned to by taskbar and have it run as an administrator. 
![Alt-Spacebar](https://github.com/Flow-Launcher/Flow.Launcher/assets/33429580/edf63124-13ec-422a-8bed-4f3af4ad7497)
![click-taskbar-icon](https://github.com/Flow-Launcher/Flow.Launcher/assets/33429580/1ebf3eee-9b88-4ea2-8d73-244d5e99195c)

**Reply by jjw24** [reply]
Top one is PowerToys Run, bottom one is Flow :) 
Because PT Run is using the alt space keys that's why you can only trigger flow by clicking on its icon.

**Reply by arbateman** [reply, marked as answer]
Brilliant! I forgot that I even had power toys installed.
thank you.
